bin/update/00-run.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -e
+
+RELEASE=$1
+UMBREL_ROOT=$2
+UMBREL_USER=$3
+
+echo
+echo "======================================="
+echo "============= OTA UPDATE =============="
+echo "======================================="
+echo "========= Stage: Pre-update ==========="
+echo "======================================="
+echo
+
+echo "Installing Umbrel $1 at $2"
+
+# Update status file
+cat <<EOF > $UMBREL_ROOT/statuses/update-status.json
+{"state": "installing", "progress": 20, "description": "Backing up"}
+EOF
+
+# Cleanup just in case there's temp stuff lying around from previous update
+echo "Cleaning up any previous backup"
+[ -d /tmp/umbrel-backup ] && rm -rf /tmp/umbrel-backup
```

Comment:
Yeah I agree, since we already nuke the backup after successful installation (in `03-run.sh`), no reason for it to exist during the next update installation. And if it does, then we can just exit.